config.yaml

```diff
@@ -40,9 +41,10 @@ commands:
         short_flag: s
   oncall:
     executable: /home/bundle/cog-command
+    description: Retrieve the current oncall for all, or a specified service
```

Comment:
Rewording suggestion: "Find out who is on-call for all services, or for a specific service"